code/src/pages/MovieList.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react'
+import { API_URL } from '../reusable/urls'
+import { Link } from 'react-router-dom'
+
+import MovieElement from '../components/MovieElement'
+
+const MovieList = ({filterType}) => {
+  const [movieList, setMovieList] = useState([])
+
+  useEffect (() => {
+   fetch(API_URL(filterType))
+      .then(response => response.json())
+      .then(data => setMovieList(data.results))
+      .catch (err => console.dirr(err))
```

Comment:
Please remember to remove all console outputs from your code before submitting. 👮🏻